db/compaction/compaction_job.cc

```diff
@@ -2745,4 +2745,194 @@ bool CompactionServiceInput::TEST_Equals(CompactionServiceInput* other,
 #endif  // NDEBUG
 #endif  // !ROCKSDB_LITE
 
+// Install the results of a remote compaction into this job's data
+void CompactionJob::RunRemote(PluggableCompactionService* service) {
```

Comment:
This change doesn't look like it's resolving merge conflicts. Where did it come from?